engine/gamesys/src/gamesys/resources/res_sound_data.cpp

```diff
@@ -378,12 +321,21 @@ namespace dmGameSystem
 
     static ResourceResult ResourceTypeSoundData_Register(HResourceTypeContext ctx, HResourceType type)
     {
-        SoundDataContext* context = new SoundDataContext;
-        context->m_Mutex = dmMutex::New();
-        context->m_CacheSize = SOUNDDATA_DEFAULT_CACHE_SIZE;
-        context->m_ChunkSize = SOUNDDATA_DEFAULT_CHUNK_SIZE;
+        // This function is called once for each file type
+        // But we only want to create one global chunk cache
+        if (g_SoundDataContext == 0)
```

Comment:
A quirk of using the same cache for all sound types, we need a global context, but the api's aren't arranged that way.